security/x509.go

```diff
@@ -133,14 +132,14 @@ func GenerateCA() ([]byte, crypto.PrivateKey, error) {
 // GenerateNodeCert generates a node certificate and returns the cert bytes as
 // well as the private key used to generate the certificate.
 // The CA cert and private key should be passed in.
-func GenerateNodeCert(caCert *x509.Certificate, caKey crypto.PrivateKey, hosts []string) (
+func GenerateNodeCert(caCert *x509.Certificate, caKey crypto.PrivateKey, keySize int, hosts []string) (
 	[]byte, crypto.PrivateKey, error) {
-	privateKey, publicKey, err := generateKeyPair()
+	privateKey, publicKey, err := generateKeyPair(keySize)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	template, err := newTemplate()
+	template, err := newTemplate("Cockroach Node")
```

Comment:
We'll most likely want "root". This will be finalized when I actually enforce client certs and link them to requested users.